nx/item.go

```diff
@@ -140,6 +141,55 @@ func extractItems(nodes []gonx.Node, textLookup []string) map[int32]Item {
 		}
 	}
 
+	searches = []string{"/Item/Consume"}
```

Comment:
Thanks for adding this

Reply:
No problem, it's added some unsupport nx options, but I can get those later.